openfisca_france/parameters/impot_revenu/plafond_qf/abat_dom/plaf_GuadMarReu.yaml

```diff
@@ -0,0 +1,6 @@
+description: Plafond maximal d'abattement DOM - Guadeloupe, Martinique, Réunion
+reference: ipp
+unit: currency
+values:
+  2001-01-01:
+    value: 5100.0
```

Comment:
il faudrait ajouter une ligne vide à la fin du fichier

Reply:
Il est en général possible de configurer son éditeur pour que ce soit fait automatiquement à la sauvegarde. ([here](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files) for vscode)